src/app/components/Disclaimer/README.md

```diff
@@ -1,5 +1,3 @@
-# Disclaimer
```

Comment:
This results in there being no header when you view the README docs e.g. https://storybook-7-upgrade--5d28eb3fe163f6002046d6fa.chromatic.com/?path=/docs/components-disclaimer--readme

It is displayed further down the page, but only because one of the stories is named "Disclaimer" 

Not quite sure what we want to do about this - keep the headers in the READMEs, and remove the stories from the README? (Which is what happens in WebCore - they have separated the docs from the stories (e.g. [Footer Docs](https://static.files.bbci.co.uk/core/storybook/index.html?path=/docs/components-navigation-global-footer--docs) and [Footer Example](https://static.files.bbci.co.uk/core/storybook/index.html?path=/story/components-navigation-global-footer--basic)

![image](https://github.com/bbc/simorgh/assets/58214768/29ea9e3c-1182-4a5f-8110-d9f2405efc9b)

Reply:
Could remove the stories from the readme and just keep the titles in readme.

Makes more sense and is just an additional click to get into the individual story anyway.

Reply:
This is a little difficult to fully control. I've added in titles to the top of the readme, but they will still appear down the page above the actual story examples. Can't quite get it to conditionally render the stories if it is a story or not if its just a readme/MDX doc.

Reply:
I might have a play around with it to see what we can do?

Reply:
Yea sure, work away.

Reply:
https://github.com/bbc/simorgh/pull/11446/commits/7a648f13653575a4b008c9dc822966f4f47306db seems to render MDX pages OK, and only the content of a Stories page (but we do sometimes end up with a blank page because we don't have docs for all components...)

Reply:
Ah cool. Thanks for taking a look. Thats something we can work with 👍

Reply:
I've tweaked it again so that it will display:

- Docs Page with Content e.g. [Disclaimer](https://storybook-7-upgrade--5d28eb3fe163f6002046d6fa.chromatic.com/?path=/docs/components-disclaimer--readme)
- Docs Page with Title Only e.g. [Caption](https://storybook-7-upgrade--5d28eb3fe163f6002046d6fa.chromatic.com/?path=/docs/components-caption--readme)
- MDX Files e.g. [Clean Code](https://storybook-7-upgrade--5d28eb3fe163f6002046d6fa.chromatic.com/?path=/docs/coding-standards-clean-code--readme)